docs/excel/excel-add-ins-ranges-unbounded.md

```diff
@@ -1,15 +1,24 @@
 ---
 title: Read or write to an unbounded range using the Excel JavaScript API
-description: Learn how to use the Excel JavaScript API to read or write to an unbounded range.
-ms.date: 02/17/2022
+description: Understand what unbounded ranges are, why cell-level operations return null or fail, and how to safely work with entire rows or columns by narrowing scope.
+ms.date: 09/22/2025
 ms.localizationpriority: medium
 ---
 
 # Read or write to an unbounded range using the Excel JavaScript API
 
-This article describes how to read and write to an unbounded range with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see [Excel.Range class](/javascript/api/excel/excel.range).
+Use these guidelines to understand how entire-column and entire-row addresses behave, and apply patterns that reduce errors and memory usage. For the complete list of properties and methods that the `Range` object supports, see [Excel.Range class](/javascript/api/excel/excel.range).
 
-An unbounded range address is a range address that specifies either entire columns or entire rows. For example:
+## Key points
+
+- "Unbounded" means entire columns (like `A:F`) or entire rows (such as `2:2`).
+- Cell-level properties (like `values`, `text`, `numberFormat`, or `formulas`) come back as `null` for unbounded reads.
+- You can't set cell-level properties on an unbounded range. This returns an error.
+- Always narrow to the used cells first.
+- Use `getUsedRange()` or intersect with a bounded range to get real cells.
```

Comment:
Can you confirm if "real" is the correct term to use here?

Reply:
Good catch! I rephrased this line.